package.json

```diff
@@ -27,51 +27,55 @@
     "typecheck": "tsc --noEmit",
     "version": "yarn build"
   },
+  "dependencies": {
+    "@manuscripts/assets": "^0.6.1",
+    "prosemirror-commands": "~1.5.0",
```

Comment:
Is it intentional to add these with a tilde?

Reply:
This was not included in the last round of updates. Fixed now.